cmd/explaintest/t/collation_misc.test

```diff
@@ -12,12 +12,25 @@ create table t(a varchar(20) charset latin1);
 insert into t values ("t_value");
 alter table t modify column a varchar(20) charset latin1;
 select * from t;
---error 8200
+
 alter table t modify column a varchar(20) charset utf8;
```

Comment:
Please add the `admin check table t`.
Add please do this for other similar places.